src/main/java/hudson/plugins/s3/callable/S3UploadCallable.java

```diff
@@ -33,9 +34,10 @@
     private final boolean produced;
     private final boolean useServerSideEncryption;
 
-    public S3UploadCallable(boolean produced, String accessKey, Secret secretKey, boolean useRole, Destination dest, List<MetadataPair> userMetadata, String storageClass,
+    public S3UploadCallable(boolean produced, String accessKey, Secret secretKey, boolean useRole, String bucketName, Destination dest, List<MetadataPair> userMetadata, String storageClass,
```

Comment:
Are we sure there is nobody using the constructor directly that will be affected by the change in method signature?